### PR TITLE
[SW-616] Propose transaction wording fixes

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/components/Modals/TransferSend.vue [hunk 1]
```diff
@@ -34,8 +34,12 @@
...
-        :text="showSendButton ? $t('pages.send.send') : $t('modals.send.next')"
+        :icon="(showSendButton && !isMultisig) ? ArrowSendIcon : null"
+        :text="showSendButton
+          ? isMultisig
+            ? $t('modals.multisigTxProposal.proposeAndApprove')
+            : $t('pages.send.send')
+          : $t('modals.send.next')"
         data-cy="next-step-button"
         @click="proceedToNextStep"
       />
```
Comment (pstachel) [inline src/popup/components/Modals/TransferSend.vue:45]: If I'm not mistaken we do have two "Send Buttons" - I'd suggest wrapping the whole logic in a component and use it in both places (Account Details & Dashboard)

File: src/popup/components/Modals/TransferSend.vue [hunk 2]
```diff
@@ -34,8 +34,12 @@
...
         class="button-action-primary"
         :disabled="error || !isConnected || !transferData.address || !transferData.amount"
-        :icon="showSendButton ? ArrowSendIcon : null"
-        :text="showSendButton ? $t('pages.send.send') : $t('modals.send.next')"
+        :icon="(showSendButton && !isMultisig) ? ArrowSendIcon : null"
+        :text="showSendButton
+          ? isMultisig
+            ? $t('modals.multisigTxProposal.proposeAndApprove')
+            : $t('pages.send.send')
+          : $t('modals.send.next')"
```
Comment (pstachel) [inline src/popup/components/Modals/TransferSend.vue:42]: Double ternary operator should be avoided, but if you are in need you should wrap it in a computed property

**Review by pstachel — CHANGES_REQUESTED** [review]

File: src/popup/components/Modals/TransferSend.vue (hunk 2 above)
Comment (pstachel) [reply src/popup/components/Modals/TransferSend.vue:42]: Or maybe you could use two separate components for it

Comment (peronczyk) [reply src/popup/components/Modals/TransferSend.vue]: I created computed property for this. Please check if it's OK.

**Review by pstachel — APPROVED** [review]

**Review by ifaouibadi — APPROVED** [review]

**Pull request merged**